nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -189,6 +189,12 @@ in
           '';
         };
       };
+
+      waylandOnly = lib.mkOption {
```

Comment:
Might also want to clarify that it does not apply to flashback, which is always X11. But it should hopefully still be compatible if one uses both, as `gnome-flashback` creates its own session files.